sdk/go/core/docsgen/api/types/ydk_types.rst

```diff
@@ -268,44 +268,172 @@ These are how YANG types are represented in Go.
 
     An interface type that represents a basic container in YANG
 
-    .. function:: GetEntityData()
+    .. function:: GetEntityData() Entity
```

Comment:
This function doesn't return Entity; it returns *CommonEntityData. The line should be
`.. function:: GetEntityData() *CommonEntityData`